create.go

```diff
@@ -260,6 +263,7 @@ func (r *Resource) computeCreateEventPatches(ctx context.Context, obj interface{
 			list, err := k8sClient.CoreV1().Nodes().List(o)
 			if tenant.IsAPINotAvailable(err) {
 				// fall through
+				r.logger.LogCtx(ctx, "level", "debug", "message", "tenant cluster is not available")
```

Comment:
This was really bad without any log, nothing we can see from logs that we fall through.